docs/data/charts/pie/webUsageStats.ts

```diff
@@ -0,0 +1,70 @@
+// Data derived from https://gs.statcounter.com/os-market-share/desktop/worldwide/2023
+// And https://gs.statcounter.com/os-market-share/mobile/worldwide/2023
+// And https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/worldwide/2023
+// For the month of December 2023
+
+export const desktopOS = [
+  {
+    label: 'Windows',
+    value: 72.72,
+  },
+  {
+    label: 'OS X',
+    value: 16.38,
+  },
+  {
+    label: 'Linux',
+    value: 3.83,
+  },
+  {
+    label: 'Chrome OS',
+    value: 2.42,
+  },
+  {
+    label: 'Other',
+    value: 4.65,
+  },
+];
+
+export const mobileOS = [
+  {
+    label: 'Android',
+    value: 70.48,
+  },
+  {
+    label: 'iOS',
+    value: 28.8,
+  },
+  {
+    label: 'Other',
+    value: 0.71,
+  },
+];
+
+export const platforms = [
+  {
+    label: 'Mobile',
+    value: 59.12,
+  },
+  {
+    label: 'Desktop',
+    value: 40.88,
+  },
+];
+
+const normalize = (v: number, v2: number) => Number.parseFloat(((v * v2) / 100).toFixed(2));
```

Comment:
Recently found that formatting method which simplify the management of digits after comma

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#style

```
new Intl.NumberFormat('de-DE', { style: 'percent', maximumFractionDigits: 2  }).format(
    number,
  ),
```

Reply:
I don't think using the number format would be better in this case, we are clipping the number number so they are easier to read, yes, but they also need to be a numeric value, while `Intl.NumberFormat` would format it to a user to see, like adding `.` in the hundreds, eg: `1000` -> `1.000`

Reply:
What triggered me is the `.toFixed(2)` before parsing.

You have do

1. number computation `(v * v2) / 100)`
2. transformation to string `.toFixed(2)`
3. transformation to number `Number.parseFloat`()

Whereas you could 
1. have the correct numbers by doing only the `(v * v2) / 100)` in the `normalize`
2. display value as expected with 

```js
const valueFormatter = new Intl.NumberFormat('de-DE', { style: 'percent', maximumFractionDigits: 2  }).format;
```

Reply:
Ah, that could be an options yes. 

I don't think it matters much though, since we also have the data as `11.11` in the other dataset, this ensures all datasets have the same precision. 😅